ngraph/core/src/op/util/rnn_cell_base.cpp

```diff
@@ -66,6 +68,59 @@ bool ngraph::op::util::RNNCellBase::visit_attributes(AttributeVisitor& visitor)
     return true;
 }
 
+void ngraph::op::util::RNNCellBase::validate_input_rank_dimension(
+    const std::vector<ngraph::PartialShape>& input)
+{
+    enum
```

Comment:
Why do you need this local enumeration?
Looks like you use only part of values from this enum.

Reply:
https://github.com/openvinotoolkit/openvino/pull/1683